### PR TITLE
Assign regions and grantees to cdi grants

### PR DESCRIPTION
**Pull request by jasalisbury**
**Description of change**

The Admin UI allows CDI grants to be assigned a region and grantee. When a CDI grant is encountered (a grant in region 13) while importing a CDI flag is set to true (and the region/grantee are not updated). The admin UI looks for this flag to allow the grant to be managed. After being assigned a grantee and region the grant can be selected in an activity report.

**How to test**

1) Pull down changes
2) Run migrations and re-seed your database
3) Go to the admin page and assign regions/grantees to CDI grants
4) Verify you can select a CDI grant in a report (you'll have to assign that grant to region 1 to have it select-able in the report, assuming your user is also assigned region 1)

or play around on [sandbox](https://tta-smarthub-sandbox.app.cloud.gov/)

**Issue(s)**
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-93

**Checklist**

- [x] Meets issue criteria
- [x] Code tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

**Review by kryswisnaskas — APPROVED** [review]
Looks good and it works!

**Review by gopar — APPROVED** [review]

**Review by rahearn — CHANGES_REQUESTED** [review]
Nice work. I have a couple of UX suggestions for dealing with the larger numbers of grants and grantees in production.

1) Please add a "Show only active" filter (default to true)
2) Please make the grantee selector be typeahead.
3) Please include the grantee's ID next to the name just so we can spot any cases where there might be two grantees that share a name.

File: src/migrations/20210412211312-add-cdi-flag-to-grants.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
 module.exports = {
-  up: async (queryInterface, Sequelize) => {
-    queryInterface.sequelize.transaction((transaction) => Promise.all([
-      queryInterface.addColumn(
+  up: async (queryInterface, Sequelize) => queryInterface.sequelize.transaction(
```
Comment (rahearn) [inline src/migrations/20210412211312-add-cdi-flag-to-grants.js:2]: 👍🏻 worked perfectly this time.

**Review by rahearn — CHANGES_REQUESTED** [review]
Sorry, I wasn't clear enough on the typeahead. Adding the type filter to the grants list is fine, so it can stay, but I was talking about The grantee <select> for the typeahead and adding the id to the display.


Screenshot of grantee selector options: https://user-images.githubusercontent.com/285842/115597830-b7186700-a2a7-11eb-9b13-c30ba2eea978.png

There may even be duplicates in that list, but it's difficult to be sure. The whole list is _far_ longer than I can display in one screenshot.


One other change to make that I forgot to mention yesterday, please add the `cdi` column to the logical data model diagram.

**Comment by rahearn** [comment]
Screenshot

I don't know why my review comment wouldn't display that screenshot

**Comment by jasalisbury** [comment]
> Sorry, I wasn't clear enough on the typeahead. Adding the type filter to the grants list is fine, so it can stay, but I was talking about The grantee for the typeahead and adding the id to the display. Screenshot of grantee selector options in next comment There may even be duplicates in that list, but it's difficult to be sure. The whole list is far longer than I can display in one screenshot. One other change to make that I forgot to mention yesterday, please add the cdi column to the logical data model diagram.

ahhh that is my bad I read `grantee` as `grant`

**Review by rahearn — APPROVED** [review]
🎉

**Pull request merged**